Agenta-AI/agenta
# fix: creates evaluators when app is created

## Description

- This PR tackles a bug in which an exact JSON match is created with each served variant.

## Changes

- This PR removes the `create_ready_to_use_evaluators()` when each variant is made and rather invokes it once when the app is made.

## Related Issues

- #1901 

## QA

- Create a first variant of an application, and check that ready-to-use evaluators were created.
- Create a second variant of the same application, an check that no new ready-to-use evaluators were created.
